src/components/work-package-details/work-package-details.test.tsx

```diff
@@ -0,0 +1,9 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from '@testing-library/react';
+import WorkPackageDetails from './work-package-details';
+
+describe('Rendering Work Packagae Summary Test', () => {});
```

Comment:
Tests? I believe Jest requires at least one `it()`/`test()` for the test suite `describe()` to pass.